testing/matrix-sdk-test/src/event_factory.rs

```diff
@@ -790,6 +806,16 @@ impl EventFactory {
         self.event(event)
     }
 
+    /// Create a new `org.matrix.msc3672.beacon` event.
```

Comment:
An example for this method would be nice, though not required.

Reply:
Sure thing, I'm happy to make the docstring more useful by adding an example, thanks for the suggestion!

Reply:
Done in fixup commit 819ee49